iml-agent/src/action_plugins/stratagem/action_cloudsync.rs

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) 2020 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+extern crate futures;
+extern crate tokio;
+
+use crate::agent_error::{ImlAgentError, RequiredError};
+use crate::lustre::search_rootpath;
+use iml_wire_types::{FidError, FidItem};
+use liblustreapi::LlapiFid;
+use tokio::process::Command;
+use tracing::error;
+
+fn get_unique(input: &str) -> String {
+    let mut output = String::new();
+    for c in input.chars() {
+        match c {
+            '/' => output.push_str("_"),
+            _ => output.push(c),
+        }
+    }
+    output
+}
```

Comment:
```suggestion
fn get_unique(input: &str) -> String {
    input.replace('/', "_")
}
```

Why not just a `replace`?